cloud: update steps for importing data (#18782)
pingcap/docs: tidb-cloud/migrate-sql-shards.md

    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.

2. Select **Import data from S3**.

    If this is your first time importing data into this cluster, select **Import From Amazon S3**.

3. On the **Import Data from Amazon S3** page, fill in the following information:

    - **Import File Count**: select **Multiple files**.
    - **Included Schema Files**: select **No**.
    - **Data Format**: select **CSV**.
    - **Folder URI** or **File URI**: fill in the bucket URI of your source data. You can use the second-level directory corresponding to tables, `s3://dumpling-s3/store/sales/` in this example, so that TiDB Cloud can import and merge the data in all MySQL instances into `store.sales` in one go.
    - **Bucket Access** > **AWS Role ARN**: enter the Role-ARN you obtained.

>     If the location of the bucket is different from your cluster, confirm the compliance of cross region.

Reviewer: Do we need to indent this line only with two spaces? Since it's the first sentence following a sub-list.
```suggestion
  If the location of the bucket is different from your cluster, confirm the compliance of cross region.
```

    TiDB Cloud starts validating whether it can access your data in the specified bucket URI. After validation, TiDB Cloud tries to scan all the files in the data source using the default file naming pattern, and returns a scan summary result on the left side of the next page. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).

4. Click **Connect**.

5. In the **Destination** section, select the target database and table.

6. Edit the CSV configuration if needed.

>    You can also click **Edit CSV configuration** to update the CSV-specific configurations, including separator, delimiter, header, `backslash escape`, and `trim last separator`.

Author: ```suggestion
    You can also click **Edit CSV configuration** to update the CSV-specific configurations, including separator, delimiter, header, `backslash escape`, and `trim last separator`.
```

>     You can also click **Edit CSV configuration** to update the CSV-specific configurations, including separator, delimiter, header, `backslash escape`, and `trim last separator`.

Author: ```suggestion
    You can also click **Edit CSV configuration** to configure Backslash Escape, Separator, and Delimiter for more fine-grained control. For more information about the CSV configuration, see [CSV Configurations for Importing Data](/tidb-cloud/csv-config-for-import-data.md).
```
